<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>780</y>
     <width>671</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact us via : @comfortablynumb7 on Telegram or 09908611517</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProfileButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>791</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company Address : Iran - Tehran - Narmak - IUST</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>971</width>
     <height>480</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>969</width>
      <height>478</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="TitleLable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>315</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="SearchBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Search Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>70</y>
     <width>111</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>121</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Runnn!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
